include/exadg/fluid_structure_interaction/precice/driver_fluid.h

```diff
@@ -218,7 +216,32 @@ class DriverFluid : public Driver<dim, Number>
 
       // post-solve
       if(is_new_time_window)
+      {
+        // computes new time-step size
         fluid->time_integrator->advance_one_timestep_post_solve();
+        // next, we synchronize the time-step sizes. Have a look in the base class for more
```

Comment:
If I understand the implementation correctly, then this comment should start with:

"The current implementation does not allow to synchronize the time step size between the fluid field and precice. Instead, the fluid field dictates the time step size. The reasons behind are that ..."

or similar. My point is to start the comment with the capabilities of the current implementation (and subsequent (optional) explanations). In the current form it is hard to understand the differences between (i) what the current implementation can do and (ii) what one would like to have/support in the future.

Reply:
I tried to improve on that. My goal is to still document current thoughts so that I understand limitations when I look into this later on.